src/streamlink/plugins/bigo.py

```diff
@@ -67,5 +70,15 @@ def _get_streams(self):
         sid, uid, ip, port = flashvars.groups()
         yield "live", BigoStream(self.session, sid, uid, ip, port)
 
+        page = http.get(self.url,
+                        allow_redirects=True,
+                        headers={"User-Agent": "Mozilla/5.0 (iPhone; CPU iPhone OS 8_0_2 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Version/8.0 Mobile/12A366 Safari/600.1.4"})
```

Comment:
You could update the other http call too as it was written before that module was added :-)